### DISCUSSION TITLE
Evaluation measures

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/MLAlgorithms#7] [Q&A] [answered]
What are the Evaluation metrics used for Regression Algorithms?

**Comment by Ganesh-Ramavath** [comment]
To evalatue the performance of Model ,that helps to understand how well our model is performed on given data
following are the regression Algorithm:
1.Mean Absolute Error
2.Mean Square Error
3.Root Mean Square Error
4.R squared

**Comment by BelureSanket** [comment, marked as answer]
Evaluation metrics used for Regression Algorithms are:
-Mean absolute error
-Mean squared error
-Root mean squared error
-R squared

**Comment by Arun3076** [comment]
Evaluation metrices are used to know how  well our ml model or algorithm is working without any errors . 
Evaluation metrices are also called as Performance measures.
The following are the evaluation metrics used for Regression Algorithms : 
1.Mean Absolute Error
2.Mean Squared Error
3. Root Mean Squared Error
4. R Squared error

**Comment by Saivarshithgithub** [comment]
Regression: is a machine learning concepts which is used to find the relationship between dependent and independent variables (still there are many definitions)
Before evaluation measure we need to what is it as the name it self tells after the model is prepared we need to find whether the prediction is done correctly or not in regression terminology we say it has best fit, bad or worst fit 
Evaluation metrics are:
Means absolute error (MAE)
Means Squared Error (MSE)
R^2
Adjusted R^2

**Comment by ThejasGajula** [comment]
Evaluation measures are metrics used to assess the performance or effectiveness of a machine learning model. 
They help us in choosing suitable algorithm for a particular problem 
Evaluation measures are
Mean absolute error
Mean square error 
Root mean square error 
R squared error

**Comment by Mazin7d4** [comment]
Evaluation metrics are used to assess the performance of the model. These metrics measure how well the predicted values from the model match the actual values. 

The most commonly used evaluation metrics for regression algorithms are:

-Mean Absolute Error (MAE)
-Mean Squared Error (MSE)
-Root Mean Squared Error (RMSE)
-R-squared (R²)

**Comment by palthyasurekha** [comment]
Evaluation metrics used for regression
Algorithms are 4 types:
-Mean absolute error
-Mean squared error
-Root mean squared error
-R squared

**Comment by lavachary** [comment]
Evaluation metrics are used to evaluate the performance of the model. These metrics helps us to understand how well our model has performed for given data.
evaluation metrics are as follows
1.Mean absolute error(MAE)
2.Mean squared error(MSE)
3.Root Mean squared error(RMSE)
R squared score
